[2025-08-25 13:00:18,600] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2025-08-19T00:00:00+00:00 [queued]>
[2025-08-25 13:00:18,630] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2025-08-19T00:00:00+00:00 [queued]>
[2025-08-25 13:00:18,630] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-25 13:00:18,630] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-08-25 13:00:18,630] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-25 13:00:18,723] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2025-08-19 00:00:00+00:00
[2025-08-25 13:00:18,725] {standard_task_runner.py:52} INFO - Started process 2795266 to run task
[2025-08-25 13:00:18,732] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'meu_primeiro_dag', 'cria_pasta', 'scheduled__2025-08-19T00:00:00+00:00', '--job-id', '453', '--raw', '--subdir', 'DAGS_FOLDER/meu_primeiro_dag.py', '--cfg-path', '/tmp/tmpwksrvmih', '--error-file', '/tmp/tmp3wuo486z']
[2025-08-25 13:00:18,732] {standard_task_runner.py:80} INFO - Job 453: Subtask cria_pasta
[2025-08-25 13:00:18,825] {task_command.py:370} INFO - Running <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2025-08-19T00:00:00+00:00 [running]> on host itoneptl
[2025-08-25 13:00:19,046] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=meu_primeiro_dag
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2025-08-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-19T00:00:00+00:00
[2025-08-25 13:00:19,047] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-25 13:00:19,047] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/root/Documents/airflow/pasta"']
[2025-08-25 13:00:19,091] {subprocess.py:85} INFO - Output:
[2025-08-25 13:00:19,092] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-25 13:00:19,256] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=meu_primeiro_dag, task_id=cria_pasta, execution_date=20250819T000000, start_date=20250825T160018, end_date=20250825T160019
[2025-08-25 13:00:19,365] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-25 13:00:19,382] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
